cfme/tests/services/test_generic_service_catalogs.py

```diff
@@ -8,7 +8,7 @@
 from cfme.services.catalogs.service_catalogs import ServiceCatalogs
 from cfme.services.catalogs.catalog_item import CatalogBundle
 from cfme.services import requests
-from cfme.exceptions import CandidateNotFound
+from widgetastic.exceptions import NoSuchElementException
```

Comment:
I think you actually want `from selenium.common.exceptions import NoSuchElementException`